src/Accounts/Views/Network/Groups.cshtml

```diff
@@ -0,0 +1,83 @@
+﻿@model CommunAxiom.Accounts.ViewModels.Network.ManageViewModel
+@{
+    ViewData["Title"] = "Manage Network";
+}
+<div class="hero is-halfheight">
```

Comment:
Page is not responsive. 
The design should also be similar across all three Network pages. Contacts uses Cards. 
Hero is typically used for media content, not for data driven content.
https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_hero_image

Reply:
implemented groups in the same way as contacts was created. page is now responsive. Initially this checkin was to publish contacts and i was going to address groups later.

I have made the necessary adjustments and implemented groups